joomla/joomla-cms
# Correct afterLoad time in Joomla Debug Console

#### Description

This PR corrects the `afterLoad` time and memory usage in the debugger `Profile Information` block.
As it is now Joomla is returning a not real page generation time because it's not counting the load time (files loaded in index.php or administrator/index.php).

Before PR:
![image](https://cloud.githubusercontent.com/assets/9630530/12274224/d5204edc-b961-11e5-91f3-ce5c92365366.png)

After PR:
![image](https://cloud.githubusercontent.com/assets/9630530/12274371/801e5c84-b962-11e5-9d44-bbbb20fc7937.png)
#### How to test
1. Apply this patch.
2. Turn on the debug in global config and the debug system plugin.
3. Test any page in frontend or admin and check if the `afterLoad` time and memory usage is in the debugger `Profile Information` block.
